src/tasks/ShopSpecialTask.py

```
@@ -0,0 +1,158 @@
+            raise
+
+    def do_run(self):
+        rounds = max(1, int(self.config.get(self.CONF_ROUNDS, 1)))
+        success_count = 0
+        failed_count = 0
+
+        self.info_set("成功次数", "0")
+        self.info_set("失败次数", 0)
+        self.info_set("失败原因", None)
+        self.info_set("当前营业额", "-")
+        self.log_info(f"开始店长特供，共 {rounds} 轮")
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)